CHANGELOG.md

```diff
@@ -364,29 +371,43 @@ _TODO_
 
 - Fixed issue with pypgstac loads which caused some writes to fail ([#18](https://github.com/stac-utils/pgstac/pull/18))
 
-[unreleased]: https://github.com/stac-utils/pgstac/compare/v0.6.10...HEAD
-[v0.6.10]: https://github.com//stac-utils/pgstac/compare/v0.6.9...v0.6.10
-[v0.6.9]: https://github.com//stac-utils/pgstac/compare/v0.6.8...v0.6.9
-[v0.6.8]: https://github.com//stac-utils/pgstac/compare/v0.6.7...v0.6.8
-[v0.6.7]: https://github.com//stac-utils/pgstac/compare/v0.6.6...v0.6.7
-[v0.6.6]: https://github.com//stac-utils/pgstac/compare/v0.6.5...v0.6.6
-[v0.6.5]: https://github.com//stac-utils/pgstac/compare/v0.6.4...v0.6.5
-[v0.6.4]: https://github.com//stac-utils/pgstac/compare/v0.6.3...v0.6.4
-[v0.6.3]: https://github.com//stac-utils/pgstac/compare/v0.6.2...v0.6.3
-[v0.6.2]: https://github.com//stac-utils/pgstac/compare/v0.6.1...v0.6.2
-[v0.6.1]: https://github.com//stac-utils/pgstac/compare/v0.6.0...v0.6.1
-[v0.6.0]: https://github.com//stac-utils/pgstac/compare/v0.5.1...v0.6.0
-[v0.5.1]: https://github.com//stac-utils/pgstac/compare/v0.5.0...v0.5.1
-[v0.5.0]: https://github.com//stac-utils/pgstac/compare/v0.4.5...v0.5.0
-[v0.4.5]: https://github.com//stac-utils/pgstac/compare/v0.4.4...v0.4.5
-[v0.4.4]: https://github.com//stac-utils/pgstac/compare/v0.4.3...v0.4.4
-[v0.4.3]: https://github.com//stac-utils/pgstac/compare/v0.4.2...v0.4.3
-[v0.4.2]: https://github.com//stac-utils/pgstac/compare/v0.4.1...v0.4.2
-[v0.4.1]: https://github.com//stac-utils/pgstac/compare/v0.4.0...v0.4.1
-[v0.4.0]: https://github.com//stac-utils/pgstac/compare/v0.3.4...v0.4.0
-[v0.3.4]: https://github.com//stac-utils/pgstac/compare/v0.3.3...v0.3.4
-[v0.3.3]: https://github.com//stac-utils/pgstac/compare/v0.3.2...v0.3.3
-[v0.3.2]: https://github.com//stac-utils/pgstac/compare/v0.3.1...v0.3.2
-[v0.3.1]: https://github.com//stac-utils/pgstac/compare/v0.3.0...v0.3.1
-[v0.3.0]: https://github.com//stac-utils/pgstac/compare/v0.2.8...v0.3.0
-[v0.2.8]: https://github.com//stac-utils/pgstac/compare/ff02c9cee7bbb0a2de21530b0aeb34e823f2e95c...v0.2.8
+[unreleased]: https://github.com/stac-utils/pgstac/compare/v0.7.10...HEAD
```

Comment:
took the liberty of updating this section which hasn't been updated since 0.6.10